widgets/src/lib/Icons/IconMagicWand.svelte

```diff
@@ -0,0 +1,22 @@
+<script>
```

Comment:
```suggestion
<script lang="ts">
```

let's take the habit of adding `lang="ts"` to our script tags, since it is becoming mandatory :)